apps/rehabstod/src/components/SickLeave/SickLeaveDegreeInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import { IDSIcon } from '@frontend/ids-react-ts'
+
+export function SickLeaveDegreeInfo({ degrees }: { degrees: number[] }) {
+  const hasSeveralDegrees = degrees.length > 1
+
+  return (
+    <div className="flex gap-1 whitespace-nowrap">
+      <span className="font-bold">{degrees[0]}%</span>
+      {hasSeveralDegrees && (
+        <>
+          <IDSIcon name="arrow" size="xs" className="my-auto" color="currentColor" color2="currentColor" />
+          {degrees[degrees.length - 1]}%
```

Comment:
According to sketches, bold should be replaced
Image from sketches:
![image](https://user-images.githubusercontent.com/79509664/233023489-dbbe3cb6-fdf1-4777-ab59-30e4f876b2b9.png)

Image from existing code:
![image](https://user-images.githubusercontent.com/79509664/233023535-72f193a3-c993-4b17-b1fd-6f25419da2ee.png)

Reply:
Should be fixed now :)